server/views/pages/induction/previousWorkExperience/workExperienceDetail.njk

```diff
@@ -0,0 +1,81 @@
+{% extends "../../../partials/layout.njk" %}
+
+{% set pageId = "induction-previous-work-experience-detail" %}
+{% set title = "What did " +  prisonerSummary.firstName + " " + prisonerSummary.lastName + " do in their " + typeOfWorkExperience | formatJobType | lower + " job?" %}
+{% set pageTitle = title %}
+
+{#
+Data supplied to this template:
+  * prisonerSummary - object with firstName and lastName
+  * backLinkUrl - url of the back link
+  * backLinkAriaText - the aria label for the back link
+  * form - form object containing the following fields:
```

Comment:
This is the new template being migrated (work experience details)
It might need a little tweaking - probably in respect of the form fields - once we decide and understand how the "mini-flow" will work, and therefore what the form object looks like.
But this is enough of a starter for 10